tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,79 @@
-import pytest
-import SCF
-
-
-def test_calc_nuclear_repulsion_energy(mol_h2o):
-    assert True
-#    assert SCF.calc_nuclear_repulsion_energy(mol_h2o) == 8.00236706181077,\
-#        "Nuclear Repulsion Energy Test (H2O) Failed"
-
-
-def test_calc_initial_density(mol_h2o):
-    """
-    Tests that the initial density returns a zero matrix
-    and tests dimensions
-    """
-
-    Duv = SCF.calc_initial_density(mol_h2o)
-    assert Duv.sum() == 0.0
-    assert Duv.shape == (mol_h2o.nao,mol_h2o.nao)
+import pytest
+import SCF
+
+
+def test_calc_nuclear_repulsion_energy(mol_h2o):
+    assert True
+#    assert SCF.calc_nuclear_repulsion_energy(mol_h2o) == 8.00236706181077,\
+#        "Nuclear Repulsion Energy Test (H2O) Failed"
+
+import pytest
+import SCF
+import main
+import pickle
+
+# Overlaps the Integrals
```

Comment:
The ability to access these is already in the fixture and you should use the fixture to access them. Putting them here, as described in class, destroys the "unit testing" aspect because now all of your tests are dependent on these rather than the functionality in the fixture.